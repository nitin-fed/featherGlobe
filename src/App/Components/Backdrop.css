.Backdrop {
  height: 100%;
  width: 100%;
  position: fixed;

  left: 0;
  top: 0;
  background-color: rgba(0, 0, 0, 0.7);
  overflow-x: hidden;
  overflow-y: auto;
}
