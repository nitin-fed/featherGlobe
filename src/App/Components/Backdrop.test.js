import React from 'react';
import Backdrop from './Backdrop';
import renderer from 'react-test-renderer';

describe('Backdrop', () => {
  test('if show === true', () => {
    const props = {
      show: true,
      onClick: jest.fn()
    };
    const component = renderer.create(<Backdrop {...props} />);
    let tree = component.toJSON();
   // expect(tree).toMatchSnapshot();
  });

  test('if show == false', () => {
    const props = {
      show: false,
      onClick: jest.fn()
    };
    const component = renderer.create(<Backdrop {...props} />);
    let tree = component.toJSON();
    //expect(tree).toMatchSnapshot();
  });
});
