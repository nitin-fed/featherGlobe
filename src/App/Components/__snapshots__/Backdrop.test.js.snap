// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Backdrop if show == false 1`] = `null`;

exports[`Backdrop if show === true 1`] = `
<div
  className="Backdrop"
/>
`;
